Sorted Fixes/sorted_fixes.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import pandas as pd
+import json
+import sys
+import os
+import datetime
+    
+#adjust base_url value if your account is on a different platform
+base_url = "https://api.kennasecurity.com"
+fixes_url = base_url + "/fixes"
+
+headers = {"Accept": "application/json", "X-Risk-Token":os.environ.get("kenna_api_key"), "User-Agent": 'sorted_fixes/1.0.0 (Kenna Security)'}
+
+sorted_fixes = pd.DataFrame()
+
+try:
+    first_page = requests.get(fixes_url, headers=headers).json()
+except ConnectionError:
+    print(f"Connection error detected. Please try again.")
+    sys.exit(1)
+
+total_count = first_page['meta']['total_count']
+total_count = int(total_count)
+page = 1
+pages = int(first_page['meta']['pages'])
+
+print(f'{total_count} fixes found.')
+
+while page <= pages:
+    fixes_loop_url = fixes_url + "?page=" + str(page)
```

Comment:
I would suggest including a `page_size` of 50 or 100 to reduce API invokes.  The default is 25.